src/EnergyPlus/InputProcessing/InputProcessor.cc

```diff
@@ -453,6 +453,30 @@ bool InputProcessor::findDefault(Real64 &default_value, json const &schema_field
     return false;
 }
 
+Real64 InputProcessor::getDefaultRealValue(std::string const &objectWord, std::string const &fieldName)
```

Comment:
@mbadams5 Please advise here.  This works perfectly fine, but it feels like it took too much to just get the default value. At a minimum, I need to decide what to do here regarding the failure state.  I should probably have this function return a bool and just return the default value on a parameter passed by reference.